packages/service-clients/odsp-client/package.json

```diff
@@ -133,7 +133,6 @@
 		"typescript": "~5.1.6"
 	},
 	"typeValidation": {
-		"disabled": true,
```

Comment:
This won't work until a version has been released with the new name. If you want to enable type tests now you'll need to manually add the previous version with the old name.

Reply:
Let's defer enabling until after a version has been released then, thanks for the heads up Tyler!

Reply:
Disabled typetests. thanks Tyler! :)